src/platform/qt/rc/io.nanoboyadvance.NanoBoyAdvance.desktop

```diff
@@ -1,12 +1,12 @@
-# Created with jdDesktopEntryEdit 1.0
 [Desktop Entry]
 Type=Application
 Name=NanoBoyAdvance
+GenericName=Emulator
```

Comment:
Isn't that too generic?

For example `Web Browser` and `File Browser` are Generic Names, so this should be `SomeThing Emulator`?

Reply:
Changed to `Game Boy Advance Emulator`, thanks!